pyproject.toml

```diff
@@ -46,7 +46,7 @@ dependencies = [
     "python-magic>=0.4.27",
     "pytz",
     "pyyaml>=5.4.1",
-    "rapyuta_io>=2.3.1",
+    "rapyuta-io @ git+https://github.com/rapyuta-robotics/rapyuta-io-sdk@c743b8f158bb0af4bc7a7322abc032d6867fe2a1",
```

Comment:
todo: Please update it to the devel commit for now. We'll replace it with the released version once SDK is released with the SDK change.